tensorflow/python/keras/activations.py

```diff
@@ -172,6 +180,13 @@ def softplus(x):
 @keras_export('keras.activations.softsign')
 def softsign(x):
   """Softsign activation function.
+  
+  Example Usage:
+  
+  >>> a = tf.constant([-20, -1.0, 0.0, 1.0, 20], dtype = tf.float32)
```

Comment:
The `-20` and `20` values don't bring anything new to this example.

Reply:
Removed the 20's